Configuration/StandardSequences/python/VtxSmeared.py

```diff
@@ -38,7 +38,9 @@
     'Shifted15mmCollision2015'  :    'IOMC.EventVertexGenerators.VtxSmearedShifted15mmCollision2015_cfi',
     'NominalCollision2015'  :        'IOMC.EventVertexGenerators.VtxSmearedNominalCollision2015_cfi',
     'NominalHICollision2015':      'IOMC.EventVertexGenerators.VtxSmearedNominalHICollision2015_cfi',
-    'ZeroTeslaRun247324Collision'  : 'IOMC.EventVertexGenerators.VtxSmearedZeroTeslaRun247324Collision_cfi'
+    'ZeroTeslaRun247324Collision'  : 'IOMC.EventVertexGenerators.VtxSmearedZeroTeslaRun247324Collision_cfi',
+    'RealisticCentroidNominalEmittance13TeVCollision': 'IOMC.EventVertexGenerators.RealisticCentroidNominalEmittance13TeVCollision_cfi',
```

Comment:
Hello Riccardo @rmanzoni ,
you should fix the path name IOMC.EventVertexGenerators.RealisticCentroidNominalEmittance13TeVCollision_cfi
to be consistent with the file you've added in IOMC
The same comment applies also to #10050 , #10049  , #10048  ,